Marlin/pins_PRINTRBOARD_REVF.h

```diff
@@ -68,6 +71,28 @@
 
 #define BOARD_NAME         "Printrboard Rev F"
 #define LARGE_FLASH        true
+#ifndef USBCON
+  #error "USBCON should be defined by the platform for this board."
+#endif
+
```

Comment:
I asked about this previously only because of issues around this historically with various instabilities surrounding USBCON (see #6550, #6737, #4171, #2433, etc)

So it sounds like it compiled correctly without this becasue the Teensyduino extension must define USBCON when the correct board is selected in the AIDE ?  If so, defining again (in case user is not using Teensyduino) certainly sounds correct.  If not, it begs the question how this is working fine currently without it.

Thanks,

-=dave